src/bindings/python/src/pyopenvino/core/tensor.cpp

```diff
@@ -14,12 +14,32 @@ namespace py = pybind11;
 
 void regclass_Tensor(py::module m) {
     py::class_<ov::Tensor, std::shared_ptr<ov::Tensor>> cls(m, "Tensor");
+    cls.doc() = "openvino.runtime.Tensor holding either copy of memory or shared host memory.";
 
     cls.def(py::init([](py::array& array, bool shared_memory) {
                 return Common::tensor_from_numpy(array, shared_memory);
             }),
             py::arg("array"),
-            py::arg("shared_memory") = false);
+            py::arg("shared_memory") = false,
```

Comment:
BTW why false is default? As i remember when we bind it we discussed that true should be default value.

Reply:
No, not at all. `False` is the default one since the begging as it is actually easier to understand and use -- there is no need to keep array alive.

Reply:
If so we also need to update this line https://github.com/openvinotoolkit/openvino/blob/master/src/bindings/python/src/openvino/runtime/ie_api.py#L42, because np.array's constructor also performs copy default and we double copy it.

Reply:
Discussed offline